drivers/ethernet/Kconfig.stm32_hal

```diff
@@ -56,6 +56,12 @@ config ETH_STM32_HAL_RX_THREAD_PRIO
 	  Preemptive scheduling can lead to more responsive handling of network traffic,
 	  especially under high load.
 
+config ETH_STM32_HAL_ETH_POST_INIT_PRIO
+	int "Second part of the ethernet initialization"
+	default 89
+	help
+		Second part of the ethernet initialization
```

Comment:
Please provide more details in help section.
Also nitpick on the help section indentation (should be 2 spaces)

Reply:
Some correction has been added regarding formatting. Also the description is now better.